### PR TITLE
Add built-in OTC CCE driver

### PR DESCRIPTION
**Pull request by outcatcher**
Add built-in driver for Open Telekom Cloud container engine

### CODE REVIEW

**Comment by maggieliu** [comment]
https://github.com/rancher/rancher/issues/30297

File: pkg/data/management/kontainerdriver_data.go [hunk 1]
```diff
@@ -101,6 +101,17 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 		return err
 	}
 
+	if err := creator.addCustomDriver(
+		"otccce",
```
Comment (dramich) [inline pkg/data/management/kontainerdriver_data.go:105]:
```suggestion
		"opentelekomcloudcloudcontainerengine",
```

**Review by dramich — COMMENTED** [review]
Update the name, otherwise this looks fine and appears to load into rancher properly.

File: pkg/data/management/kontainerdriver_data.go (hunk 1 above)
Comment (outcatcher) [reply pkg/data/management/kontainerdriver_data.go:105]: I suppose I will change it to `opentelekomcloudcontainerengine`, same as above drivers

**Review by dramich — APPROVED** [review]

**Comment by dramich** [comment]
@outcatcher Thanks for the PR! Can you open a backport pr against the `release/v2.5` branch?

**Pull request merged**